integration/kubernetes/runtimeclass_workloads/vfio.yaml

```diff
@@ -8,6 +8,7 @@ kind: Pod
 metadata:
   name: vfio
 spec:
+  terminationGracePeriodSeconds: 0
```

Comment:
@devimc Why are you adding this and killing the pod before it can exit gracefully? 
Maybe using a `kubectl wait` to wait for the pod to be terminated could help.

Reply:
because all the yaml files under `integration/kubernetes/runtimeclass_workloads/vfio.yaml ` have it, for example:

https://github.com/kata-containers/tests/blob/0a48afb93f27a385aa8b0bb7d72e5cd4bc6b32d5/integration/kubernetes/runtimeclass_workloads/busybox-pod.yaml#L11

Reply:
Ok. We should address this separately then.